RELEASES.md

```diff
@@ -1,3 +1,13 @@
+# 0.7.0
+### Added
```

Comment:
Sorry my comment was a bit confusing probably: I meant that we should use this naming convention, but currently we're not using that. So it's better to keep consistent the naming and then refactoring the whole changelog (unless you want to refactor now). So what I'm suggesting is, use the present here as well as it was done in the other part of the same file; but we should in the future, when we have formalized in the way the changelog needs to be done, be more consistent with the format suggested in https://keepachangelog.com/en/1.0.0/.

So, my suggestion would be rename `Added` to `Add`, `Removed` to `Remove` and `Changed` to `Change`. Unless you want to rename also the past section as well – that would be fine too.
The important thing is the consistency.